example/example.html

```diff
@@ -1,13 +1,22 @@
 <!DOCTYPE html>
 <html ng-app="demoApp">
   <body>
-    <main>
-      <ngc-omnibox></ngc-omnibox>
+    <main ng-controller="OmniboxExampleController">
```

Comment:
Are we using controllerAs in a later pull request? I don't want to embarrass ourselves on the open source world by still using `$scope` attachments.